test/cropharvest/test_full_integration.py

```diff
@@ -0,0 +1,41 @@
+from cropharvest.datasets import CropHarvest
+from cropharvest.inference import Inference
+from pathlib import Path
+from sklearn.ensemble import RandomForestClassifier
+
+DATA_DIR = "data"
+TIF_FILE = Path(__file__).parent / "98-togo_2019-02-06_2020-02-01.tif"
+
+
+def test_full_integration():
```

Comment:
I appreciate the tests you've added 😍

Reply:
I have to do it because I break code too easily 🙈